preproc_script_1.sh

```diff
@@ -125,28 +133,39 @@ do
 	fslstats -K ./"$workingdir"/mc_qc/EPI_snr_mask.nii.gz ./"$workingdir"/mc_qc/EPI_mc_tSNR.nii.gz -n -m > ./"$workingdir"/mc_qc/EPI_mean_tSNR.txt
 	rm ./"$workingdir"/mc_qc/EPI_snr_mask.nii.gz
 	rm ./"$workingdir"/mc_qc/EPI_mc_std.nii.gz
-	rm ./"$workingdir"/mc_qc/EPI_mc_mean.nii.gz
+#    rm ./"$workingdir"/mc_qc/EPI_mc_mean.nii.gz
```

Comment:
What's the reason for removing this line? Replacing `N4BiasFieldCorrection -d 3 -i ./"$workingdir"/EPI_topup_mean.nii.gz` with `N4BiasFieldCorrection -d 3 -i ./"$workingdir"/EPI_c_mean.nii.gz` during brain extraction?

Reply:
-because ./"$workingdir"/mc_qc/EPI_mc_mean  is called again in Ln163
-It was already replaced in the script (?)
N4BiasFieldCorrection -d 3 -i ./"$workingdir"/EPI_c_mean.nii.gz -o ./"$workingdir"/EPI_n4.nii.gz -c [100x100x100,0] -b [50] -s 2